ui/env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_GOOGLE_CLIENT_ID=694013380015-j2lgv2uucaqs6shl72km6unmq77v74fn.apps.googleusercontent.com
```

Comment:
why is this changed?

Reply:
eh.. why is the .env uploaded, it should be in the .gitignore? ?